account_invoice_triple_discount/tests/test_invoice_triple_discount.py

```diff
@@ -164,3 +183,26 @@ def test_04_discounts_decimals_tax(self):
         invoice_form.save()
 
         self.assertEqual(invoice.amount_tax, 177.61)
+
+    def test_05_refund_negative_taxes(self):
```

Comment:
I tried this locally and this test is passing even without the changes to `account_move`.
When adding a fix, the included test should pass with the fix and break without it.

Reply:
@SirTakobi now I fixed this test and without our fix in _recompute_tax_lines it fails